EVGEN/AliGenExtFile.cxx

```diff
@@ -112,9 +112,9 @@ void AliGenExtFile::Generate()
   //
   Double_t origin[3] = {0,0,0};
   Double_t time = 0.;
-  Double_t p[4];
+  Double_t p[4] = {0,0,0,0};
```

Comment:
OK, but why not 

    Double_t p[] = {0.,0.,0.,0.};

?

Reply:
Agreed, I just followed the same way of coding of the previous lines, I can remove it.